src/openshift/openshift-pipeline-template.yml

```diff
@@ -36,29 +36,15 @@ parameters:
 
 ##
 # Objects created by the template:
-# * An image stream containing the application
 # * A build config using the pipeline strategy
-# * A deployment config / replication controller instantiating 1 instance of the application
-# * A service to make the application available to other application running in OpenShift
-# * A route making the application accessible from outside OpenShift
```

Comment:
This file is not used anymore, it's going to be removed as the launchpad generates it.